telethon/_updates/messagebox.py

```diff
@@ -828,4 +828,4 @@ def end_channel_difference(self, request, reason: PrematureEndReason, chat_hashe
         else:
             raise RuntimeError('Unknown reason to end channel difference')
 
-    # endregion Getting and applying channel difference.
+    # endregion Getting and applying channel difference.
```

Comment:
Seems this newline was accidentally removed.

```suggestion
    # endregion Getting and applying channel difference.

```